perception/probabilistic_occupancy_grid_map/launch/laserscan_based_occupancy_grid_map.launch.py

```diff
@@ -108,6 +108,7 @@ def launch_setup(context, *args, **kwargs):
                         "input_obstacle_and_raw_pointcloud"
                     ),
                 },
+                {"updater_type": LaunchConfiguration("updater_type")},
```

Comment:
If you want to  define `updater_type` in launch configuration, you should change `<updator>.yaml` file structure.

For example, [binary_bayes_filter_updater.param.yaml](https://github.com/soblin/autoware.universe/blob/fix/occupancy-grid-missing-arg/perception/probabilistic_occupancy_grid_map/config/binary_bayes_filter_updater.param.yaml) should be look like:

```yaml
/**:
  ros__parameters:
    probability_matrix:
      occupied_to_occupied: 0.95
      occupied_to_free: 0.05
      free_to_occupied: 0.2
      free_to_free: 0.8
    v_ratio: 10.0
```

You also need to change [updater.cpp file](https://github.com/soblin/autoware.universe/blob/fix/occupancy-grid-missing-arg/perception/probabilistic_occupancy_grid_map/src/updater/occupancy_grid_map_binary_bayes_filter_updater.cpp):

```cpp
void OccupancyGridMapBBFUpdater::initRosParam(rclcpp::Node & node)
{
  probability_matrix_(Index::OCCUPIED, Index::OCCUPIED) =
    node.declare_parameter<double>("binary_bayes_filter.probability_matrix.occupied_to_occupied");
  probability_matrix_(Index::FREE, Index::OCCUPIED) =
    node.declare_parameter<double>("binary_bayes_filter.probability_matrix.occupied_to_free");
  probability_matrix_(Index::FREE, Index::FREE) =
    node.declare_parameter<double>("binary_bayes_filter.probability_matrix.free_to_free");
  probability_matrix_(Index::OCCUPIED, Index::FREE) =
    node.declare_parameter<double>("binary_bayes_filter.probability_matrix.free_to_occupied");
  v_ratio_ = node.declare_parameter<double>("binary_bayes_filter.v_ratio");
}
```

## Todo summary

- change updater.yaml structure in package and autoware_launch
- change updater initROSparam function in updater .cpp file